sema/check/service.py

```
@@ -0,0 +1,60 @@
+# your_submodule/service.py
+
+import os
+import yaml
+from .testing.base import TestBase
```

Ruff

`.testing.base.TestBase` imported but unused

Remove unused import: `.testing.base.TestBase`

(F401)